src/ProgressBar.js

```diff
@@ -74,10 +74,10 @@ export default class ProgressBar {
 	}
 
 	installProgressElement() {
-		this.progressElement.style.width = '0';
+		this.progressElement.style.width = '0%';
 		this.progressElement.style.opacity = '1';
 		document.documentElement.insertBefore(this.progressElement, document.body);
-		this.refresh();
+    this.setValue(Math.random() * 0.4);
```

Comment:
0.4 is a bit of a jump sometimes, I've found something around 0.25 a bit smoother.